templates/preproc_template.sh.j2

```diff
@@ -11,6 +11,12 @@
 {% endif %}
 ###Create *files file###
 mothulity_fc {{files_directory}} -o {{files_directory}}{{job_name}}.files
+{% if dbcut_settings %}
+###Prepare database###
+mothur '#set.current(processors={{processors}});
+pcr.seqs(fasta={{dbcut_settings.org_db}}, start={{dbcut_settings.region_start}}, end={{dbcut_settings.region_end}}, keepdots=F)'
+mv {{dbcut_settings.cut_out}} {{align_database}}
```

Comment:
This overwrites the database downloaded by the user, without even informing.

Reply:
It doesn't, during the execution of main motulity script I assign the path to original database to org_db variable, while the align_database value was replaced by the new database name. 
Note that database used as an input for pcr (/cutting) is dbcut_settings[org_db].
I did it to be able to avoid creating an entire new template just for one parameter change. 
On the other hand there is a risk that user somehow have a file named <DB_version>_<region_coordinates>.align so maybe the warning should be placed in description of the customize_region option. What do you think?

Reply:
I believe that a user re-using a trimmed database is aware enough and would just specify it in a regular way. Since there is only an option to trim supplied during the main execution, the trimmed database should be just temporary.

Reply:
I do not quite get it about moving/renaming. I agree that passing a proper value to align_database is a way to go. But why do this in the template file? Why not just set the ```align_database``` variable to ```dbcut_settings.cut_out``` content? We should keep the logic in the ```Python``` part rather than templates.

Reply:
OK, now I see your point. The comment above is no longer relevant

Reply:
Ok, Im not changing anything then :)